modules/system/combat.js

```diff
@@ -129,8 +127,7 @@ export default class CombatHelpers {
         msg += CombatHelpers.checkSizeFearTerror(combat)
 
         if (msg)
-            ChatMessage.create({ content: msg })
-
+            await ChatMessage.create({ content: msg })
```

Comment:
Doesn't seem necessary to await this (and no reason for the function to be async)